code/game/objects/effects/spawners/ghostroles.dm

```diff
@@ -0,0 +1,70 @@
+//Prisoner containment sleeper: Ghosts become escaped prisoners and are advised to find a way out of the mess they've gotten themselves into.
+/obj/effect/mob_spawn/human/prisoner_transport
+	name = "prisoner containment sleeper"
+	desc = "A sleeper designed to put its occupant into a deep coma, unbreakable until the sleeper turns off. This one's glass is cracked and you can see a pale, sleeping face staring out."
+	mob_name = "an escaped prisoner"
+	icon = 'icons/obj/Cryogenic2.dmi'
+	icon_state = "sleeper_1"
+	outfit = /decl/hierarchy/outfit/escapedprisoner
+	short_desc = "You're a prisoner, sentenced to hard work in one of Ironhammer's labor camps, but it seems as \
```

Comment:
`Ironhammer`

Reply:
We'll do a lore pass later. I'd rather not change it in this file